ui/package.json

```diff
@@ -23,6 +23,7 @@
     "@angular/platform-browser-dynamic": "^4.0.0",
     "@angular/router": "^4.0.0",
     "core-js": "^2.4.1",
+    "enhanced-resolve": "^3.3.0",
```

Comment:
I believe this line should be:
`"enhanced-resolve": "~3.3.0",`
rather than the above, because `^3.3.0` will be comfortable with the broken version (which is 3.4.0)